plugins/guests/redhat/cap/rhn_register.rb

```diff
@@ -19,7 +19,7 @@ def self.rhn_register(machine)
         def self.rhn_register_register(machine, ui)
           rhn_register_upload_certificate(machine, ui)
           rhn_register_server_url(machine, ui) if machine.config.registration.serverurl
-          command = "rhnreg_ks #{configuration_to_options(machine.config.registration)}"
+          command = "/usr/sbin/rhnreg_ks #{configuration_to_options(machine.config.registration)}"
```

Comment:
cz with "sudo" (for example as vagrant-user) "rhnreg_ks" is not in the path of the vagrant-user.
With full path it works perfectly for root directly and vagrant via sudo - at least for me.